CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.10.0 (14/05/2021)
+
+**olp-sdk-dataservice-write**
+
+- Added the `MultiPartUploadWrapper` class. You can use it to upload large files in a browser and Node.js to the Blob API v1 and v2 services in chunks.
+- Changed `VersionedLayerClient`. Now, to upload large files to versioned layers, use the `MultiPartUploadWrapper` class.
+- Added the abstracted `BlobData` class. It is a wrapper of data (`Blob`, `Buffer`, `File`, string, and so on)
+  that allows to read a specific range of bytes and get data size.
+
+**olp-sdk-dataservice-api**
+
+- Fixed the issue with the incorrect `body` parameter in the `ObjectApi.startMultipartUploadByKey` function.
+
+**olp-sdk-core**
+
+- Added `export * as fs from "fs";` to the Node.js index file and changed typing settings from `index.web` to `index`.
```

Comment:
Probably it is enough to say: `Added export for fs symbols to the Node.js index file....`